subpackages/assertions/source/unit_threaded/assertions.d

```diff
@@ -47,7 +47,7 @@ void shouldBeFalse(E)(lazy E condition, string file = __FILE__, size_t line = __
  * Verify that two values are the same.
  * Throws: UnitTestException on failure
  */
-void shouldEqual(V, E)(scope auto ref V value, scope auto ref E expected, string file = __FILE__, size_t line = __LINE__)
```

Comment:
I don't see what's wrong with having `scope` here?

Reply:
It could still be inferred, but this functions calls `formatValueInItsOwnLine` on `value` which calls `convertToString` which might call `value.toString` which might not be `scope`.